.eslintrc

```diff
@@ -32,16 +32,16 @@
     "no-unused-vars": 0, // Rely on typescript-eslint rule above
     "@typescript-eslint/explicit-function-return-type": ["error", { "allowExpressions": true }],
     "@typescript-eslint/typedef": ["error", { "parameter": true }],
+    "@typescript-eslint/consistent-type-imports": "warn",
```

Comment:
In the original TFE PR, I ended up using the "inline-type-imports" fix style so types and imports stay on the same line. Did you decide against that stylistically?

https://github.com/allen-cell-animated/timelapse-colorizer/pull/786/files#diff-e7e195d3d8fa5b82fa51ed952f3a179d429cbc46c0432ce1f7b357c54c162822

Reply:
I would feel good about `import` and `import type` always being on separate lines for fast readability. but not a big deal either way